src/localization/en.js

```diff
@@ -116,15 +116,27 @@ export default {
       predictions:     'Predictions',
     },
 
+    formHelpers: {
+      type:    ' Type',
+      weekly:  'Weekly',
+      monthly: 'Monthly',
+      yearly:  'Yearly',
+      yes:     { label: 'Yes' },
+      no:      { label: 'No' },
```

Comment:
We could just make these `yesLabel: 'Yes'` and `noLabel: 'No'`, for simplicity. I do think having 'label' in there is a good idea though, to emphasize that we would only be putting it in the `Radio`'s `label` prop and not `value`.

Reply:
First, why is there an object instead of just a word? Is this in the model of `label` and `hint`? I'm trying to think if that makes sense considering we're not going to have a hint for these. Is there an argument for consistency here?

Second, I agree that the key may be missing something, but I'm not sure about `Label`. Do we make that a standard for all labels? If not, why here and not for others?

It's occurred to me that, in general, having the key of the snippet be the same as the text of the snippet seems not quite right. Does it only describes the contents of the snippet, not the purpose? That would seems fragile. What if the contents changes? Will we need to change the key? Or is it that way because that really is the most succinct and clear way to describe the purpose of the snippet?

I know it can sound purely philosophical to some people, but I've found that when variable names go wrong, they make the code almost unreadable and untraceable.

Reply:
I think "consistency" could actually be confusing here. The "labels" here are not doing the same job as the labels on an instance of a form component; they're really just part of the radio button control, and there's a separate label which describes what the radio buttons are for. We don't have to put `label` in the key. I don't have a strong opinion on what the keys should be; could be something involving `true` and `false` rather than `yes` and `no`, maybe.

Reply:
Hmm, what's the rationale behind `true` and `false`?

Reply:
It corresponds the value of the boolean that the radio button controls. That should never change. 

(Maybe something like `trueLabel` and `falseLabel`? Why shouldn't we use the word "label" again?)

Reply:
I hear what you're saying. That does correspond with the radio button value, but these snippets aren't for the value, they're for the label, so they don't actually represent the values `true` and `false`.

I wasn't necessarily saying to not use the word 'label'. It was more of a question about how we're going to be deciding keys in general. Other labels don't have 'label' in their name, though, honestly, other labels' keys are `label`. If it helps, it's like naming your variable `cooridnateArray` or something - labeling types instead of purpose - but I've always looked at it as a general guideline, not a rule. I think if you can't find anything better and that kind of language clarifies what's going on, than that's what you use.

Just to check in here, maybe just `yes` and `no` are fine. I never stopped to ask why those weren't sufficient. What was the downside?

Reply:
I'm fine with `yes` and `no`, but you said

>It's occurred to me that, in general, having the key of the snippet be the same as the text of the snippet seems not quite right.

so I was trying to think of other names. If the snippet is so short that it would also work well as a key, I think it makes sense to use it as a key, as long as the meaning is clear.

Reply:
Yeah, this might be a case where the purpose and the text are the same. As for it being confused for the value instead of the label, maybe that's a good point. I'd assumed the coder using the snippets would be aware of how the snippets should only be used for visible text, but maybe that's not a great assumption. Don't know why I didn't consider that before. Hmm, you have some good points.

Reply:
You know what, forget everything I said. Well, don't forget it. The discussion about purpose was one I found useful. Where I'm at right now, though, it looks like your proposal of `yesLabel` and `noLabel` sounds like the way to go.

Reply:
OK, you had good points too! But I do still like `yesLabel` and `noLabel`. So @ValerieKenyon, could you please change lines 124 and 125 to
```
yesLabel: 'Yes',
noLabel: 'No',
```
Thanks!

Reply:
Thanks but actually lines 124 and 125 are still not quite right. We want them to just be strings - `'Yes'` and `'No'` rather than `{ label: 'Yes' }` and `{ label: 'No' }`. Could you change that?

Reply:
So sorry @ethanbb , I just fixed it.